src/angular-app/languageforge/lexicon/settings/configuration/field-unified-view.model.ts

```diff
@@ -600,7 +630,7 @@ export class ConfigurationFieldUnifiedViewModel {
 }
 
 export class Group {
-  show: boolean = false;
+  show: boolean = true;
```

Comment:
Why change the default to `true` here? I don't see the reasoning behind this change.

Reply:
@palmtreefrb didn't provide an explanation in his PR or commit, but I am pretty sure this is a desired change.  We want the default to be "true" so that when an admin adds a new configuration column for a user, all of the boxes are checked.  It would seem to me that the sensible default is "true" and the admin can remove field visibility one at a time if that is desired.  I think the current state of "false" is less desired because it is likely to require more clicking than the default "true" state.